app/main/views/frameworks.py

```diff
@@ -1087,6 +1087,9 @@ def contract_review(framework_slug, agreement_id):
 @main.route('/frameworks/<framework_slug>/contract-variation/<variation_slug>', methods=['GET', 'POST'])
 @login_required
 def view_contract_variation(framework_slug, variation_slug):
+    """
+    This view asks suppliers to agree to a framework variation and then generates a confirmation email  when they do.
```

Comment:
I'd add a massive `TODO: create a variation template in Notify before adding a variation to a framework in the API` here.

Reply:
Could use detailing here too:
https://alphagov.github.io/digitalmarketplace-manual/contract-variations.html?highlight=variation